src/services/results-calculator.ts

```diff
@@ -54,6 +56,7 @@ export default class ResultsCalculator {
 	}
 
 	sortResults () {
+
 		const totalScores = map(
```

Comment:
With reference to the comment I made above about creating a new class simply for formatting the text, this last map function could be placed in that class, as it is only responsible for formatting the output of the results. This would make the 'sortResults' function more modular (and testable, as the only thing it should be doing is sorting results).